tests/unit/vdom.ts

```diff
@@ -2708,4 +2711,39 @@ describe('vdom', () => {
 			});
 		});
 	});
+
+	describe('behavior', () => {
+		it('will append nodes with attributes already attached', function(this: Test) {
```

Comment:
I think this can use a param rather than `this` for skipping (and you shouldn't need to explicitly type `Test`), like:

```ts
it('will append nodes with attributes already attached',(test) => {
    if (!global.window || !global.window.hasOwnProperty('MutationObserver')) {
        test.skip('MutationObserver not present');
    }
    // ...
}
```